pkg/monitoring/metrics/virt-controller/vmsnapshot.go

```diff
@@ -0,0 +1,41 @@
+package virt_controller
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	io_prometheus_client "github.com/prometheus/client_model/go"
+	snapshotv1 "kubevirt.io/api/snapshot/v1beta1"
+)
+
+var (
+	vmSnapshotMetrics = []operatormetrics.Metric{
+		VmSnapshotCreationTimestamp,
+	}
+
+	VmSnapshotCreationTimestamp = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vmsnapshot_timestamp_seconds",
+			Help: "Unix timestamp of when the VM snapshot was created. Including the VM name, namespace, and snapshot name.",
+		},
+		[]string{"vm_name", "vm_namespace", "snapshot_name", "snapshot_namespace", "snapshot_uid"},
+	)
+)
+
+func NewVMSnapshotCreated(snapshot *snapshotv1.VirtualMachineSnapshot, content *snapshotv1.VirtualMachineSnapshotContent) {
+	if snapshot.Status != nil && snapshot.Status.CreationTime != nil {
+		VmSnapshotCreationTimestamp.WithLabelValues(
+			content.Spec.Source.VirtualMachine.Name,
```

Comment:
@machadovilaca This is failing `make test`.. //pkg/storage/snapshot:go_default_test  failed